WordPress-Core/ruleset.xml

```diff
@@ -17,200 +17,102 @@
-
-	<!-- Covers rule: ... while spaces can be used mid-line for alignment. -->
-	<rule ref="Universal.WhiteSpace.DisallowInlineTabs"/>
-
-	<!-- Implied through the examples: align the assignment operator in a block of assignments. -->
-	<rule ref="Generic.Formatting.MultipleStatementAlignment">
-		<properties>
-			<property name="maxPadding" value="40"/>
-		</properties>
-	</rule>
-
-	<!-- Implied through the examples: align the double arrows. -->
-	<rule ref="WordPress.Arrays.MultipleStatementAlignment">
-		<properties>
-			<property name="maxColumn" value="60"/>
-		</properties>
+	<rule ref="Squiz.PHP.EmbeddedPhp.SpacingAfter">
+		<severity>0</severity>
 	</rule>
 
 
 	<!--
 	#############################################################################
-	Handbook: PHP - Brace Style.
-	Ref: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style
+	Handbook: PHP - No Shorthand PHP Tags.
+	Ref: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#no-shorthand-php-tags
 	#############################################################################
 	-->
-	<!-- Covers rule: Braces shall be used for all blocks. -->
-	<rule ref="Squiz.ControlStructures.ControlSignature"/>
-
-	<!-- Covers rule: Braces should always be used, even when they are not required. -->
-	<rule ref="Generic.ControlStructures.InlineControlStructure"/>
+	<!-- Covers rule: Never use shorthand PHP start tags. Always use full PHP tags. -->
+	<rule ref="Generic.PHP.DisallowShortOpenTag"/>
+	<rule ref="Generic.PHP.DisallowAlternativePHPTags"/>
 
 
 	<!--
 	#############################################################################
-	Handbook: PHP - Declaring Arrays.
-	Ref: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#declaring-arrays
+	Handbook: PHP - Single and Double Quotes.
+	Ref: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#single-and-double-quotes
 	#############################################################################
 	-->
-	<!-- Covers rule: Arrays must be declared using long array syntax. -->
-	<rule ref="Universal.Arrays.DisallowShortArraySyntax"/>
-
+	<!-- Covers rule: Use single and double quotes when appropriate.
+		 If you're not evaluating anything in the string, use single quotes. -->
+	<rule ref="Squiz.Strings.DoubleQuoteUsage.NotRequired"/>
 
-	<!--
-	#############################################################################
-	Handbook: PHP - Use elseif, not else if.
-	Ref: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#use-elseif-not-else-if
-	#############################################################################
-	-->
-	<!-- Covers rule: ... use elseif for conditionals. -->
-	<rule ref="PSR2.ControlStructures.ElseIfDeclaration"/>
+	<!-- Rule: Text that goes into attributes should be run through esc_attr().
+		 https://github.com/WordPress/WordPress-Coding-Standards/issues/527 -->
```

Comment:
```suggestion
	<!-- Rule: Text that goes into HTML or XML attributes should be escaped so that
		 single or double quotes do not end the attribute value.
		 https://github.com/WordPress/WordPress-Coding-Standards/issues/527 -->
```